### DISCUSSION TITLE
unable to reduce screen size (zoom out)

### DISCUSSION THREAD

**Discussion by jasjw** [getgauge/taiko#2559] [General]
**Describe the bug**
unable to reduce screen by doing the following operation :
press(["Control","-"])

**To Reproduce**
Steps (or script) to reproduce the behavior:
1. Go to website
2. press(["Control","-"])
3. no changes

**Comment by saikrishna321** [comment]
@jasjw The only way I can see this happening currently is by `evaluate(() => document.body.style.zoom = 1.0`

Set the value to the required zoom:

`1.0` is 100%
`1.1` is 110%
and so on
